articles/active-directory-b2c/page-layout.md

```diff
@@ -18,10 +18,18 @@ ms.subservice: B2C
 
 Page layout packages are periodically updated to include fixes and improvements in their page elements. The following change log specifies the changes introduced in each version.
 
-[!INCLUDE [active-directory-b2c-public-preview](../../includes/active-directory-b2c-public-preview.md)]
-
 ## Self-asserted page (selfasserted)
 
+**2.1.1**
+
+- Added a UXString `heading` in addtion to `intro` to show on page as a title, it will be hidden by default.  
```

Comment:
I recommend the following to make this clearer:
Added a UXString heading in addtion to intro to show on page as a title. This will be hidden by default